sprint-mission-7/queries.sql

```diff
@@ -0,0 +1,182 @@
+/*
+  다음 경우들에 대해 총 14개의 SQL 쿼리를 작성해 주세요.
+  예시로 값이 필요한 경우 적당한 값으로 채워넣어서 작성하면 됩니다. 
+*/
+
+/*
+  1. 내 정보 업데이트 하기
+  - 닉네임을 "test"로 업데이트
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+UPDATE
+  `User`
+SET
+  nickname = 'test'
+WHERE
+  id = 1;
+
+/*
+  2. 내가 생성한 상품 조회
+  - 현재 로그인한 유저 id가 1이라고 가정
+  - 최신 순으로 정렬
+  - 10개씩 페이지네이션, 3번째 페이지
+*/
+SELECT
+  *
+FROM
+  `Product`
+WHERE
+  userId = 1
+ORDER BY
+  createdAt DESC
+LIMIT 10 OFFSET 20;
+
+/*
+  3. 내가 생성한 상품의 총 개수
+  - 현재 로그인한 유저 id가 1이라고 가정
+*/
+SELECT
+  COUNT(*) AS '상품의 총 갯수'
```

Comment:
틀린거는 아니지만, 실무에서는 주로 영어로 별칭을 짓는 경우가 많습니다.
```suggestion
  COUNT(*) AS 'product_total'
```